kanidmd/score/src/https/mod.rs

```diff
@@ -153,17 +157,22 @@ pub fn to_tide_response<T: Serialize>(
 }
 
 // Handle the various end points we need to expose
-async fn index_view(_req: tide::Request<AppState>) -> tide::Result {
+async fn index_view(req: tide::Request<AppState>) -> tide::Result {
     let mut res = tide::Response::new(200);
-
+    eprintln!(
+        "index_view domain_display_name {}",
+        req.state().domain_display_name
+    );
     res.set_content_type("text/html;charset=utf-8");
+
+    // TODO: #860 pull the domain display name and write it here
     res.set_body(r#"
     <!DOCTYPE html>
     <html lang="en">
         <head>
             <meta charset="utf-8"/>
             <meta name="viewport" content="width=device-width">
-            <title>Kanidm</title>
+            <title>===DOMAIN_DISPLAY_NAME===</title>
```

Comment:
I think there has to be a better way than this, so maybe we leave this as is for now, and we talk about how a better method could look?

Reply:
Tweaked it, this might be a little less terrible? I might be able to take the WASMHASH bit out of the middleware now I know how to mess around with  app state.